src/repositories/raw_subscriber.py

```diff
@@ -0,0 +1,27 @@
+
+import dataclasses
+import json
+import logging
+import os
+import requests
+
+# An endpoint that handles unprocessed JSON forwarded from notifications.
+class RawSubscriber:
+    def __init__(self, url_endpoint):
+        self._url_endpoint = url_endpoint
+
+    def post_commit_status(self, commit_status):
+        json_data = dataclasses.asdict(commit_status[1])
+        logging.debug("Sending raw json to subscriber: " + json.dumps(json_data))
+        response = requests.post(url=self._url_endpoint, json=json_data)
+
+class RawSubscriberFactory:
+    @staticmethod
+    def new_raw_subscribers() -> list[RawSubscriber]:
+        # TODO(tcare): Dynamically pick up subscribers via CRD or similar
+        # Currently we only have one subscriber
+        subscribers = []
+        subscriber = RawSubscriber(os.getenv("SPEKTATE_ENDPOINT"))

```

Comment:
If we don't have a list of subscribers defined in a ConfigMap (e.g. https://github.com/kaizentm/kfp-event-handler/blob/b918e4f4f5635166e1d8c9eba8583aeb6ce5a583/src/main.go#L84 and https://github.com/kaizentm/kfp-event-handler/blob/b918e4f4f5635166e1d8c9eba8583aeb6ce5a583/kfp-event-handler.yaml#L76), let's name at least the endpoint in a generic way (e.g. CUSTOM_COMMIIT_STATE_ENDPOINT rather than SPEKTATE_ENDPOINT, so we don't have dependencies on that)

Reply:
I'll post a follow up PR for this soon